week-2-practise/level-2/4-slice.js

```diff
@@ -5,7 +5,8 @@
 // - remove the item at the specified index
 
 function remove(arr, index) {
-  return; // complete this statement
+  arr.splice(index, 1);
+  return arr; // complete this statement
```

Comment:
Note that the instructions above say that the function must not change the original array. This means you can't use `splice`, as it does change the array it's called on.